missie.md

```diff
@@ -1,41 +1,15 @@
-#Nederlands
+I *Make things*,  
+I work hard, and start things,
+I dance, sing, and drink to beautiful music.
+I either let go of my worries, or I take responsibility.
 
-Ik daag mezelf uit om mooie nuttige dingen te maken,
-en om nieuwe skills te vergaren. 
+I do the above with friends and family 
+but mostly with my wife and children, 
+for whom I provide and protect.
```

Comment:
Whom I provide for and protect. Klinkt overigens wel alsof Judith erg veel bescherming nodig heeft, weet niet of dat uitmaakt.